README.md

```diff
@@ -36,10 +36,10 @@ _Latest development version:_
 - Gallery based on [PhotoSwipe v5](https://github.com/PhotoboothProject/PhotoSwipe)
 - Standalone Gallery based on PhotoSwipe v5 ([localhost/gallery.php](http://localhost/gallery.php)).
 - Standalone Slideshow based on PhotoSwipe v5 ([localhost/slideshow](http://localhost/slideshow)).
-- Access login via [localhost/login](http://localhost/login) instead [localhost/login.php](http://localhost/login.php).
+- Access login via [localhost/login](http://localhost/login) instead [localhost/login](http://localhost/login).
```

Comment:
Remove the `instead [localhost/login.php](http://localhost/login.php).` completely (was old Location some years ago).